src/connectors/argentMobile/modal/starknet/adapter.ts

```diff
@@ -123,7 +129,18 @@ export class StarknetAdapter
       throw new Error("No session")
     }
 
-    const requestToCall = this.handleRequest[call.type]
+    let type = call.type as string
+
+    if (
```

Comment:
minor: maybe worth adding a comment that we are adding it temporarily for backwards compatibility and mobile will add support for `starknet_*` calls that we can switch to in few months

Reply:
oh yep